dhcpv6/option_bootfileparam_test.go

```diff
@@ -0,0 +1,31 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testBootfileParam = "\x00\x30initrd=http://myserver.mycompany.local/initrd.xz\x00\x0eroot=/dev/sda1\x00\x02rw\x00\x24netconsole=..:\000:.something\000here.::.."
+)
+
+func TestOptBootFileParam(t *testing.T) {
+	expected := testBootfileParam
+	opt, err := ParseOptBootFileParam([]byte(expected))
```

Comment:
This doesn't really test the multi-option parsing code, it only fills in a single string even though your example is clearly intended to expand into multiple strings

Reply:
Added multi-option test to `TestParsedTypeOptBootFileParam`